### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joshlk**
Added example as discussed in #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,36 @@ Warning: LFS client uses HTTP basic authentication, so using **HTTPS** is a must
...
+cd repo
+
+# Add a normal file
+touch test.txt
+git add test.txt
+git commit -m "normal file"
+
+# Add a lfs file 
+git lfs install
+mkfile -n 10m test.bin
```
Comment (artemkin) [inline README.md]: mkfile is not available on linux

dd is more universal way to create a file

dd if=/dev/zero of=test.bin count=10240 bs=1024

File: README.md [hunk 2]
```diff
@@ -57,6 +57,36 @@ Warning: LFS client uses HTTP basic authentication, so using **HTTPS** is a must
...
+git clone ....
+cd repo
+
+# Add a normal file
+touch test.txt
+git add test.txt
+git commit -m "normal file"
+
+# Add a lfs file 
+git lfs install
```
Comment (artemkin) [inline README.md]: git lfs command is not available by default, you have to install git lfs client first. Maybe need to add a note about it.

File: README.md [hunk 3]
```diff
@@ -57,6 +57,36 @@ Warning: LFS client uses HTTP basic authentication, so using **HTTPS** is a must
...
+mkfile -n 10m test.bin
+git lfs track test.bin
+git add .gitattributes test.bin
+git commit -m "lfs file"
+
+# Configure lfs remote to local server
+git config -f .lfsconfig lfs.url http://localhost:5000
```
Comment (artemkin) [inline README.md]: lfs server listens 8080 port by default.

File: README.md [hunk 4]
```diff
@@ -57,6 +57,36 @@ Warning: LFS client uses HTTP basic authentication, so using **HTTPS** is a must
...
+git add .lfsconfig
```
Comment (artemkin) [inline README.md]: Don't forget to commit .lfsconfig. When you clone the repo, git lfs will find this config and download lfs files automatically.

**Review by artemkin — CHANGES_REQUESTED** [review]
Added a couple of issues to be addressed before merging.

Thank you for your work!

**Review by joshlk — COMMENTED** [review]
Cheers 👍 . Changes are in

**Review by artemkin — APPROVED** [review]

**Comment by artemkin** [comment]
@joshlk Perfect! Thank you

**Pull request merged**